pkg/controller/kieapp/constants/constants.go

```diff
@@ -292,6 +292,10 @@ const (
 	SUBCOMPONENT_TYPE_INFRA = "infrastructure"
 
 	DefaultDatagridUsername = "infinispan"
+
+	KIE_ADMIN_CREDENTIALS_SECRET = "kie-admin-credentials"
+	USERNAME_ADMIN_SECRET_KEY    = "username"
+	PASSWORD_ADMIN_SECRET_KEY    = "password"
```

Comment:
The format of the credentials secret, or what keys are used, is not mentioned anywhere. So it would be good to add it to the desciption in KieApp.
Also it would be handy to add within this PR an example of credentials secret.

Reply:
ok